src/site/content/en/mini-apps/what-are-mini-apps/index.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: What are mini apps?
+authors:
+  - thomassteiner
+date: 2021-01-25
+# updated: 2021-01-25
+description: |
+  This chapter introduces the concept of mini apps and provides examples of their look and feel.
+tags:
+  - mini-apps
+---
+
+## Building blocks and compatibility
+
+Mini apps are small (commonly less than 4MB) apps that require a super app to run. What they have in
+common, independent of the super app, is that they are built with ("dialects" of) the Web
+technologies HTML, CSS, and JavaScript. The runtime of a mini app is a
+[WebView](https://research.google/pubs/pub46739/) in the super app, not the underlying operating
+system, which means mini apps are cross platform. What this means is that the same mini app can run
+in the same super app, no matter if the super app runs on Android, iOS, or another OS. However, not
+all mini apps can run in all super apps, more on this [later](/mini-app-standardization/).
+
+## Discovery
+
+Mini apps are often discovered _ad-hoc_ via branded 2d barcodes, which solves an important
+offline-to-online challenge, for example, getting from a physical restaurant menu to a payment mini
+app, or from a physical e-scooter to a rental min app. The image below depicts an example of such a
+branded 2d barcode. It is for
+[WeChat's demo mini app](https://github.com/wechat-miniprogram/miniprogram-demo). When the code is
+scanned with the WeChat super app, the mini app launches directly. Super apps other than in this
+case WeChat typically will not recognize the barcode.
```

Comment:
The last sentence doesn't make any sense.

Reply:
Rephrased.